corehq/apps/reports/templates/reports/filters/form_app_module_drilldown.html

```diff
@@ -2,110 +2,91 @@
-            <select class="form-control"
-                    id="{{ css_id }}-{{ unknown.slug }}_xmlns"
-                    name="{{ slug }}_{{ unknown.slug }}_xmlns"
-                    data-bind="options: all_unknown_forms,
-                    optionsText: 'text', optionsValue: 'val',
-                    optionsCaption: caption_text, value: selected_unknown_form"></select>
+    <div class="well well-sm {{ css_id }}-unknown_controls" data-bind="visible: is_unknown_shown">
+        <div class="form-group">
+            <label class="control-label col-xs-4 col-md-2" for="{{ css_id }}-{{ unknown.slug }}_xmlns">
+                {% trans 'Choose Unknown Form' %}
+            </label>
+            <div class="col-xs-8 col-md-10">
+                <select class="form-control"
+                        id="{{ css_id }}-{{ unknown.slug }}_xmlns"
+                        name="{{ slug }}_{{ unknown.slug }}_xmlns"
+                        data-bind="options: all_unknown_forms,
+                        optionsText: 'text', optionsValue: 'val',
+                        optionsCaption: caption_text, value: selected_unknown_form"></select>
+            </div>
         </div>
     </div>
-</div>
-{% endif %}
-<div id="{{ css_id }}-known_control">
-    {{ block.super }}
-</div>
+    {% endif %}
+    <div id="{{ css_id }}-known_control">
+        {{ block.super }}
+    </div>
 
-{% if unknown_available or display_app_type %}
-<div id="{{ css_id }}-advanced-options" style="margin-top: -.8em;">
-    <div class="checkbox">
-        <label>
-            <input name="show_advanced" type="checkbox" data-bind="checked: show" />
-            {% trans "Show Advanced Options" %}
-        </label>
+    {% if unknown_available or display_app_type %}
+    <div id="{{ css_id }}-advanced-options" style="margin-top: -.8em;">
+        <div class="checkbox">
+            <label>
+                <input name="show_advanced" type="checkbox" data-bind="checked: show" />
+                {% trans "Show Advanced Options" %}
+            </label>
+        </div>
     </div>
-</div>
-{% endif %}
+    {% endif %}
 
-{% if hide_fuzzy.show %}
-<div class="alert alert-warning {{ css_id }}-unknown_controls"
-     data-bind="visible: show"
-     style="margin-top: 1em;">
-    {% include 'reports/filters/partials/fuzzy_checkbox.html' %}
+    {% if hide_fuzzy.show %}
+    <div class="alert alert-warning {{ css_id }}-unknown_controls"
+        data-bind="visible: show"
+        style="margin-top: 1em;">
+        {% include 'reports/filters/partials/fuzzy_checkbox.html' %}
+    </div>
+    {% endif %}
 </div>
-{% endif %}
-{% endblock %}
-
-{% block filter_js %}
-<script>
-
- $.when(
-     $.getScript("{% static 'reports/js/report_filter.drilldown_options.js' %}"),
-     $.getScript("{% static 'reports/js/report_filter.advanced_forms_options.js' %}")
- ).done(function () {
-
-     // This is copied from drilldown_options.html because the order matters
```

Comment:
I think this comment might be worth preserving, after looking back at the history (http://manage.dimagi.com/default.asp?231773)